From 526bac50511aab8cb5301448dfa1e169fbe55298 Mon Sep 17 00:00:00 2001
From: Juri Lelli <juri.lelli@gmail.com>
Date: Thu, 25 Oct 2012 12:11:09 -0700
Subject: [PATCH 39/48] sched: remove commented lines in futex/rtmutex

A previous commit leaved useless commented lines in kernel/futex.c
and kernel/rtmutex.c. Just remove them.
---
 kernel/futex.c   |    2 --
 kernel/rtmutex.c |    2 --
 2 files changed, 4 deletions(-)

diff --git a/kernel/futex.c b/kernel/futex.c
index 283d874..20ef219 100644
--- a/kernel/futex.c
+++ b/kernel/futex.c
@@ -2297,8 +2297,6 @@ static int futex_wait_requeue_pi(u32 __user *uaddr, unsigned int flags,
 	 */
 	debug_rt_mutex_init_waiter(&rt_waiter);
 	rt_waiter.task = NULL;
-	//rb_init_node(&rt_waiter.tree_entry);
-	//rb_init_node(&rt_waiter.pi_tree_entry);
 
 	ret = get_futex_key(uaddr2, flags & FLAGS_SHARED, &key2, VERIFY_WRITE);
 	if (unlikely(ret != 0))
diff --git a/kernel/rtmutex.c b/kernel/rtmutex.c
index f6a9074..7187c50 100644
--- a/kernel/rtmutex.c
+++ b/kernel/rtmutex.c
@@ -720,8 +720,6 @@ rt_mutex_slowlock(struct rt_mutex *lock, int state,
 	int ret = 0;
 
 	debug_rt_mutex_init_waiter(&waiter);
-	//rb_init_node(&waiter.tree_entry);
-	//rb_init_node(&waiter.pi_tree_entry);
 
 	raw_spin_lock(&lock->wait_lock);
 
-- 
1.7.9.5

